README.md

```diff
@@ -49,36 +63,166 @@ const { MediaContextProvider, Media, createMediaStyle } = createMedia({
+export const mediaStyle = ExampleAppMedia.createMediaStyle()
+export const { Media, MediaContextProvider } = ExampleAppMedia
+```
+
+Create a new `App` file which will be the launching point for our application:
+
+```tsx
+// App.tsx
+
+import React from "react"
+import { Media, MediaContextProvider } from "./Media"
+
+export const App = () => {
+  return (
     <MediaContextProvider>
-      <Media at='sm'>
-        <MobileApp />
-      </Media>
-      <Media at='md'>
-        <TabletApp />
-      </Media>
-      <Media greaterThanOrEqual='lg'>
-        <DesktopApp />
-      </Media>
+      <Media at="xs">Hello mobile!</Media>
+      <Media greaterThan="xs">Hello desktop!</Media>
     </MediaContextProvider>
-  </>
-)
+  )
+}
 ```
 
-### Demo
+Mount `<App />` on the client:
+
+```tsx
+// client.tsx
 
-You can find an example in this repository and run it locally like so:
+import React from "react"
+import ReactDOM from "react-dom"
+import { App } from "./App"
 
+ReactDOM.render(<App />, document.getElementById("react"))
 ```
-git clone https://github.com/artsy/react-responsive-media.git
-cd react-responsive-media
-yarn install
-yarn example
-open http://localhost:8080
+
+Then on the server, setup SSR rendering and pass `mediaStyle` into a `<style>`
+tag in the header:
+
+```tsx
+// server.tsx
+
+import React from "react"
+import ReactDOMServer from "react-dom/server"
+import express from "express"
+
+import { App } from "./App"
+import { mediaStyle } from "./Media"
+
+const app = express()
+
+app.get("/", (_req, res) => {
+  const html = ReactDOMServer.renderToString(<App />)
+
+  res.send(`
+    <html>
+      <head>
+        <title>@artsy/fresnel - SSR Example</title>
+
+        <!–– Inject the generated styles into the page head -->
+        <style type="text/css">${mediaStyle}</style>
+      </head>
+      <body>
+        <div id="react">${html}</div>
+
+        <script src='/assets/app.js'></script>
+      </body>
+    </html>
+  `)
+})
+
+app.listen(3000, () => {
+  console.warn("\nApp started at http://localhost:3000 \n")
+})
 ```
 
-### Why not conditionally render?
+And that's it! To test, disable JS and scale your browser window down to a
+mobile size; it will correctly render without the need to use a user-agent or
+other server-side "hints".
+
+## Usage with Gatsby
+
+`@artsy/fresnel` works great with Gatsby's static hybrid approach to rendering.
+See the [Gatsby Example](`/examples/gatsby`) for a simple implementation.
+
+## Example Apps
+
+There are three examples one can look explore in the `/examples` folder:
+
+- [Basic]('/examples/basic)
+- [Server-side Rendering]('/examples/ssr-rendering)
+- [Gatsby]('/examples/gatsby)
+- [Kitchen Sink]('/examples/kitchen-sink)
+
+While the `Basic` and `SSR` examples will get one pretty far, `@artsy/fresnel`
+can do _a lot_, and can be used as a tool to build other tools. For an
+exhaustive deep-dive into its features, check out the `Kitchen Sink` app.
```

Comment:
I’m not sure what you mean by “[it] can be used to build other tools”. What in the kitchen sink example are you referring to? And perhaps it’s best to link to the specific part of the example?